src/backend/Dockerfile.graphQlApi

```diff
@@ -0,0 +1,68 @@
+FROM maven:3.6.3-jdk-8 as build
+
+ARG STARTERS_V
+
+ENV STARTERS_V=${STARTERS_V}
+
+# Install bcgov springbootstarter
+
+RUN mkdir /root/.ssh
+RUN chmod 0700 /root/.ssh 
+RUN ssh-keyscan -t rsa localhost >> ~/.ssh/known_hosts
+
+RUN apt-get install -y git
+
+WORKDIR /libs
+
+RUN git clone https://github.com/bcgov/spring-boot-starters.git
+
+WORKDIR /libs/spring-boot-starters
+
+RUN git checkout $STARTERS_V
+
+RUN mvn install -P all -f src/pom.xml
+
+# Install efiling-hub libs
+
+WORKDIR /
+
+COPY . .
+
+RUN mvn -B clean install -P efilng-graphql
+
+WORKDIR /efiling-graphql
+
+# Build quarkus api
+
+RUN mvn package -Dquarkus.package.type=fast-jar
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3 
+
+ARG JAVA_PACKAGE=java-11-openjdk-headless
```

Comment:
This should be 8 for now. Unless you want to make the split. 
ARG JAVA_PACKAGE=java-1.8.0-openjdk-headless

Reply:
good point, it is running with 11 now, I wonder if we should give it a try...